code/modules/vehicles/vehicle.dm

```diff
@@ -35,8 +35,8 @@
 		. = ..()
 
 
-/obj/vehicle/user_buckle_mob(mob/living/M, mob/user)
-	if(user.incapacitated())
+/obj/vehicle/user_buckle_mob(mob/living/M, mob/living/user)
+	if(!istype(user) || user.incapacitated())
```

Comment:
If this sanity is called by another proc that also has sanity, is this sanity insanity?

Reply:
No because buckling does not need to originate in click code.

Reply:
So then it is an insanity superposition!

Reply:
wouldn't it be better to have the sanity check only here then?

Reply:
If we had responsible developers yes, but i'll settle for redundancy here for peace of mind.